assets/react/components/profile.js

```diff
@@ -15,29 +17,64 @@ export default class MyPosts extends Component {
 	previewPost(post){
 		this.context.router.transitionTo('/posts/' + post.id + '/preview');
 	}
+	update_name(name){
+		this.props.dispatch(updateUser(this.props.profile_user, {name: name}))
+	}
+	update_email(email){
+		this.props.dispatch(updateUser(this.props.profile_user, {email: email}))
+	}
+	getProfileSection(){
+		if(!this.props.user){
+			return ""
+		}
+		var elems = []
+		elems.push(<img className="p-image" src={this.props.profile_user.photo} />)
+		if(this.props.profile_user.admin){
+			elems.push(<div className="p-admin" > Admin </div>)
+		}
+		if( this.props.user && this.props.user.admin || (this.props.user.id == this.props.profile_user.id)){
```

Comment:
Dont think we should add admin here. I dont see a use case.
And again..how id this.props.user.id working?